_includes/neurips/2020/about.neurips.html

```diff
@@ -349,10 +349,10 @@ <h2 style="text-align: center">Organizers</h2>
 
       <h3>Workshop Chairs</h3>
       <ul>
-        <!-- <li><em>General & Logistics Co-Chairs --</em> Maria Luisa Santiago (Accel AI)</li> -->
+        <li><em>General & Logistics Co-Chairs --</em> Maria Luisa Santiago (Accel AI)</li>
         <li><em>Program Committee Co-Chairs --</em> Ariel Ruiz-Garcia, Ivan Arraut(The Open University of Hong Kong), Juan M. Banda (Georgia State University)</li>
         <li><em>Sponsor & Finance Co-Chairs --</em> Ignacio Lopez-Francos (NASA Ames Research Center) and Fabian Latorre (EPFL Lausanne)</li>
-        <li><em>Mentoring and Presentation Chair --</em> Pedro Magalhães Braga (Universidade Federal de Pernambuco), Karla Caballero Barajas (Pandora/Sirius XM), Sergio Hernan Garrido Mejia, Eduardo Ulises Moya-Sánchez (Gobierno del estado de Jalisco, México), Vinicius Fernandes Caridá (Itaú-Unibanco & GDE)</li>
+        <li><em>Mentoring and Presentation Chair --</em> Pedro Magalhães Braga (Universidade Federal de Pernambuco), Karla Caballero Barajas (Pandora/Sirius XM), Sergio Hernan Garrido Mejia (Technical University of Denmark/Amazon), Eduardo Ulises Moya-Sánchez (Gobierno del estado de Jalisco, México), Vinicius Fernandes Caridá (Itaú-Unibanco & GDE)</li>
         <li><em>Volunteer & Mentoring Chair --</em> Carlos Miranda (Huawei) </li>
```

Comment:
Please change this role to say "Volunteer Chair", remove Mentoring from this line.